apps/table-frontend/src/app/components/effective.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import { getFile } from './get-file-info';
+import {
+  ParsedArticleParagraphStandalone,
+  ArticleParagraphBacksUpClaim,
+} from '@foodmedicine/interfaces';
+export interface EffectiveProps {
+  fileName: string;
+}
+
+interface Effectivity {
+  effective: string;
```

Comment:
what are the options for this string? are there values you are expecting it to take?

Reply:
Oh ok sure, I added an enum